.github/actions/setup-sentry/action.yml

```diff
@@ -7,8 +7,8 @@ inputs:
     default: 'false'
   kafka:
     description: 'Is kafka required?'
-    required: false
-    default: 'false'
+    required: true
+    default: 'true'
```

Comment:
So is kafka required to run *all* test suites now? Or can we selectively turn it on when needed per suite?

Reply:
I think we'll end up adding a clear "switch" between local and distributed mode, and dist mode will have Kafka/Zookeeper up in its specific config. This was an experiment on my end, checking if this solves the issue (it did not), so it will be removed.